pkg/engine/git_test.go

```diff
@@ -120,9 +121,10 @@ func BenchmarkGitEngine(b *testing.B) {
 	defer cancel()
 
 	e, err := Start(ctx,
-		WithConcurrency(1),
+		WithConcurrency(uint8(runtime.NumCPU())),
```

Comment:
What effect did this have on the benchmarks?

Reply:
I believe the only difference was that running w/ 1 was slower but the proportions were about the same. I was also trying to simulate normal use case where we use the `runtime.NumCPU` to determine concurrency unless overridden by the user.